### DISCUSSION TITLE
Difference between 0.3.12 helm version vs. 1.3.2 application version

### DISCUSSION THREAD

**Discussion by yhp353134** [apache/superset#17393] [Q&A / Help]
superset  有 0.3.12 版本开始的  也有1.3.2的版本  请问0开头的版本  与1开头的版本有什么区别？

**Comment by bappleug-tw** [comment]
As far as I understand:

0.3.12 is the helm version for Kubernetes deployment:
https://superset.apache.org/docs/installation/running-on-kubernetes

1.3.2 is the real application version
